backend/models/office_hours/oh_ticket.py

```diff
@@ -0,0 +1,38 @@
+from pydantic import BaseModel
+from datetime import datetime
+from ticket_type import TicketType
+from ticket_state import TicketState
+
+__authors__ = ["Sadie Amato, Bailey DeSouza, Meghan Sun, Maddy Andrews"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class OfficeHoursTicketDraft(BaseModel):
+    """
+    Pydantic model to represent an `OfficeHoursTicket` that has not been created yet.
+
+    This model is based on the `OfficeHoursTicketEntity` model, which defines the shape
+    of the `OfficeHoursTicket` database in the PostgreSQL database.
+    """
+
+    office_hours_event_id: int
+    description: str
+    type: TicketType
+    state: TicketState
```

Comment:
Curious, I was unable to find these files! Is this in another branch?

Reply:
The `office-hours` branch will have these enum + entities files, and this was the PR that had initialized it: https://github.com/unc-csxl/csxl.unc.edu/pull/295

We forgot to merge those entities and enums into this current working branch; we just merged those new changes, so those files should exist now on this branch!